### PR TITLE
Added: broken and violating add-on template

### PR DESCRIPTION
**Pull request by basrieter**
### Description

As discussed I added two templates for issues for plugins: 
- Report a broken add-on
- Report a violating add-on

Testing can be done here: https://github.com/basrieter/repo-plugins/issues

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/broken_addon.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+about: Report an add-on that is currently broken within Kodi.  
+title: ''
+labels: 'Broken-Addon'
+assignees: ''
+
+---
+
+
+### Add-on details:
+<!--- Provide some details of te broken add-on. -->
```
Comment (chewitt) [inline .github/ISSUE_TEMPLATE/broken_addon.md:12]: "the" broken add-on .. same error in the other template too

Comment (basrieter) [reply .github/ISSUE_TEMPLATE/broken_addon.md:12]: Thanks! I read it and read it, but still.

File: .github/ISSUE_TEMPLATE/broken_addon.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name: Report a broken add-on
+about: Report an add-on that is currently broken within Kodi.  
+title: ''
+labels: 'Broken-Addon'
+assignees: ''
+
+---
+
+
+### Add-on details:
```
Comment (enen92) [inline .github/ISSUE_TEMPLATE/broken_addon.md:11]: maybe add a checkbox where the requester can select whether he is the author of the addon? We need to check if either way, so feel free to ignore.

Comment (basrieter) [reply .github/ISSUE_TEMPLATE/broken_addon.md:11]: Good point. I added a check-box.

**Pull request merged**